docs/scos/dev/glue-api-guides/202204.0/decoupled-glue-infrastructure/how-to-guides/how-to-create-or-change-a-convention.md

```diff
@@ -126,3 +129,28 @@ class CustomConventionPlugin extends AbstractPlugin implements ConventionPluginI
     }
 }
 ```
+
+## Use a convention
+To use newly created convention you must add it to the `GlueApplicationDependencyProvider` to the method `getConventionPlugins()`.
+
+```php
+<?php
+
+namespace namespace Pyz\Glue\GlueApplication;
+
+use Pyz\Glue\GlueJsonApiConvention\Plugin\GlueApplication\CustomConventionPlugin;
+use Spryker\Glue\GlueApplication\GlueApplicationDependencyProvider as SprykerGlueApplicationDependencyProvider;
+
+class GlueApplicationDependencyProvider extends SprykerGlueApplicationDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Glue\GlueApplicationExtension\Dependency\Plugin\ConventionPluginInterface>
+     */
+    protected function getConventionPlugins(): array
+    {
+        return [
+            new CustomConventionPlugin(),
+        ];
+    }
+}
+```
```

Comment:
Mb we can add some paragraph with info about when we have no convention in place?